pkg/monitoring/mapping.go

```diff
@@ -155,6 +157,53 @@ func MakeTransmissionMapping(
 	return out, nil
 }
 
+func MakeTelemetryConfigSetMapping(
+	envelope StateEnvelope,
+	feedConfig FeedConfig,
+) (map[string]interface{}, error) {
+	state := envelope.State
+	offchainConfig, err := parseOffchainConfig(state.Config.OffchainConfig.Raw[:state.Config.OffchainConfig.Len])
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse OffchainConfig blob from the program state: %w", err)
+	}
+	signers, err := json.Marshal(extractSigners(state.Oracles))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse signers: %w", err)
+	}
+	transmitters, err := json.Marshal(extractTransmitters(state.Oracles))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse transmitters: %w", err)
+	}
+	s, err := json.Marshal(int32ArrToInt64Arr(offchainConfig.S))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse s: %w", err)
+	}
+
+	oracles, err := createTelemetryOracles(offchainConfig.OffchainPublicKeys, offchainConfig.PeerIds, state.Oracles)
+	if err != nil {
+		return nil, fmt.Errorf("failed to oracles s: %w", err)
+	}
+
+	out := map[string]interface{}{
+		"config_digest":      base58.Encode(state.Config.LatestConfigDigest[:]),
+		"block_number":       uint64ToBeBytes(envelope.BlockNumber),
+		"signers":            signers,
+		"transmitters":       transmitters,
+		"f":                  int32(state.Config.F),
+		"delta_progress":     uint64ToBeBytes(offchainConfig.DeltaProgressNanoseconds),
+		"delta_resend":       uint64ToBeBytes(offchainConfig.DeltaResendNanoseconds),
+		"delta_round":        uint64ToBeBytes(offchainConfig.DeltaRoundNanoseconds),
+		"delta_grace":        uint64ToBeBytes(offchainConfig.DeltaGraceNanoseconds),
+		"delta_stage":        uint64ToBeBytes(offchainConfig.DeltaStageNanoseconds),
+		"r_max":              int64(offchainConfig.RMax),
+		"s":                  s,
+		"oracles":            oracles,
+		"feed_state_account": base58.Encode(feedConfig.StateAccount[:]),
+	}
+	return out, nil
```

Comment:
I just noticed that there is no test for this mapping. Can you please add one! See mapping_test.go for an example